### ISSUE TITLE
Can not install, no matching package found

### ISSUE THREAD

**Issue by tienvx** [thePanz/messenger-filesystem-transport#2]
`symfony/serializer-pack` library only have 2 versions atm: 1.0.0 and 1.0.1, but composer.json require version ^4.1, which does not match any of 2 versions above

Here is the complete error message:
```
composer require pnz/messenger-filesystem-transport --dev                                                                                                                                 12:28:19
Using version ^0.1.1 for pnz/messenger-filesystem-transport
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Installation request for pnz/messenger-filesystem-transport ^0.1.1 -> satisfiable by pnz/messenger-filesystem-transport[v0.1.1].
    - pnz/messenger-filesystem-transport v0.1.1 requires symfony/serializer-pack ^4.1 -> no matching package found.

Potential causes:
 - A typo in the package name
 - The package is not available in a stable-enough version according to your minimum-stability setting
   see <https://getcomposer.org/doc/04-schema.md#minimum-stability> for more details.
 - It's a private package and you forgot to add a custom repository to find it

Read <https://getcomposer.org/doc/articles/troubleshooting.md> for further common problems.

Installation failed, reverting ./composer.json to its original content.
```

**Comment by tienvx** [comment]
see #3

**Comment by thePanz** [comment]
Thanks! It was fixed in 6325c59.
Please, try the v0.1.2 release

**Comment by tienvx** [comment]
Awesome!
